resources/views/home/user.blade.php

```diff
@@ -0,0 +1,129 @@
+                    </p>
+                </div>
+
+                <div class="osu-page-header__box osu-page-header__box--status osu-page-header__box--graph">
+                    <div class="osu-page-header__status osu-page-header__status--fade-in">
+                        <div class="osu-page-header__status-label">
+                            Games
+                        </div>
+                        <div class="js-forum-topic-watch--unread osu-page-header__status-text">
+                            {{$currentGames}}
+                        </div>
+                    </div>
+                    <div class="osu-page-header__status osu-page-header__status--selected osu-page-header__status--fade-in osu-page-header__status--animation-delay">
+                        <div class="osu-page-header__status-label">
+                            {{trans('home.user.header.stats.online')}}
+                        </div>
+                        <div class="js-forum-topic-watch--unread osu-page-header__status-text">
+                            {{$currentOnline}}
+                        </div>
+                    </div>
+                    <div class="js-fancy-graph fancy-graph" data-src="#banchostats"></div>
+                    <script id="banchostats" type="application/json">{!! json_encode($graphData) !!}</script>
+                </div>
+            </div>
+        </div>
+
+        <div class="user-home">
+            <div class="user-home__news">
+                <h2 class="user-home__news-title">{{trans('home.user.news.title')}}</h2>
+                @if (!empty($news))
+                    <div class="user-home__news-posts">
+                        @foreach ($news as $post)
+                            @if ($loop->iteration > 3)
+                                @break
+                            @endif
+
+                            @include('home._user_news_post_preview', ['post' => $post, 'collapsed' => false])
+                        @endforeach
+                    </div>
+                    <div class="user-home__news-posts user-home__news-posts--collapsed">
+                        @foreach ($news as $post)
+                            @if ($loop->iteration <= 3)
+                                @continue
+                            @endif
+                            @if ($loop->iteration > 8)
+                                @break
+                            @endif
+
+                            @include('home._user_news_post_preview', ['post' => $post, 'collapsed' => true])
+                        @endforeach
+                    </div>
+                @else
+                    <div class="news-post-preview__post-content">
+                        <p>{{trans('home.user.news.error')}}</p>
+                    </div>
+                @endif
+            </div>
+            <div class="user-home__right-sidebar">
+                <div class="user-home__buttons">
+                    @include('home._user_giant_button', [
```

Comment:
the button doesn't seem to follow flyte's design? (the design I'm looking at is taller and doesn't take up full width)

![](https://s.0paste.com/2017-03/chrome_2017-03-28_16-17-12.png)